shared/chat/selectable-big-team-channel-container.js

```diff
@@ -23,15 +22,14 @@ const mapStateToProps = (state, {conversationIDKey}) => {
 const mapDispatchToProps = () => ({})
 
 const mergeProps = (stateProps, dispatchProps, ownProps) => {
-  const isSelected = ownProps.isSelected && !Styles.isMobile
   return {
     channelname: stateProps.channelname,
-    isSelected,
+    isSelected: ownProps.isSelected,
```

Comment:
This can't just be changed back like this. If you really want this behavior just add another prop to control what it does on mobile, I don't want the inbox selection stuff to get worse to support this feature.

Reply:
As I said in the comment below, I couldn't try the chat search out on master. What happens when you had this?

Reply:
I'm gonna try to fix the chat search on master

Reply:
@mmaxim Turns out I just needed to `gobuild`. I don't it's the right answer to override `isSelected` in this component as this is the correct "selected" style. I pushed another commit to get the same behavior by doing the mobile check in `chat/inbox-search/index.js`. PTAL thanks!